src/esbuild.mjs

```diff
@@ -147,7 +147,7 @@ async function main() {
 		plugins,
 		entryPoints: ["extension.ts"],
 		outfile: "dist/extension.js",
-		external: ["vscode", "sqlite3"],
+		external: ["vscode"], // kilocode_change - removed sqlite3
```

Comment:
sorry , my bad, sqlite3 wasn't from roo, so this kilocode_change is not necessary.